Sources/Common/Core/Math/index.js

```diff
@@ -1202,22 +1209,22 @@ export function diagonalize3x3(a_3x3, w_3, v_3x3) {
 
 export function singularValueDecomposition3x3(a_3x3, u_3x3, w_3, vT_3x3) {
   let i;
-  const B = [createArray(3), createArray(3), createArray(3)];
+  const B = createArray(9);
```

Comment:
should not be needed to zero initialize: `const B = []` is enough (or `const B = new Array(9)`)